src/control/jobs/control_jobs.c

```diff
@@ -1872,33 +1918,52 @@ static int32_t dt_control_datetime_job_run(dt_job_t *job)
 
   const guint total = g_list_length(t);
 
-  const char *mes11 = offset ? N_("adding time offset to %d image") : N_("setting date time to %d image");
-  const char *mes12 = offset ? N_("adding time offset to %d images") : N_("setting date time to %d images");
+  const char *mes11 = offset ? N_("adding time offset to %d image") : N_("setting date/time to %d image");
```

Comment:
This should read "setting date/time **of** %d image". The "adding time offset to..." is fine. Similarly below

Reply:
Can I disagree here ? :smile: 
Adding `time offset to %d image` is less correct than `setting date/time to %d image`.
Time offset should added to date/time of an image and not to an image.
If **of** is OK, `setting date/time to %d image` doesn't sound really wrong (the image gets a date/time).

Reply:
You can disagree if you like but "setting date/time **to** 1 image" is wrong. If you want to say "setting time/date of 1 image to DD/MM/YYYY" that's fine.

Reply:
you could say "applying date/time to %d image" and that would be ok

Reply:
> Don't ask me to explain why, 

I won't, every language has its own secrets. :smile:
I'll change to **of**.